content/source/docs/enterprise/admin/logging.html.md

```diff
@@ -1,86 +1,303 @@
 ---
 layout: "enterprise"
-page_title: "Logging - Infrastructure Administration - Terraform Enterprise"
+page_title: "Log Forwarding - Infrastructure Administration - Terraform Enterprise"
 ---
 
-# Terraform Enterprise Logs
+# Terraform Enterprise Log Forwarding
 
-This document contains information about interacting with Terraform Enterprise logs. There are two types of logs, application logs and audit logs. Application logs emit information about the services that comprise Terraform Enterprise. Audit logs emit information whenever any resource managed by Terraform Enterprise is changed.
+Terraform Enterprise supports forwarding its logs to one or more external
```

Comment:
Should we include a version qualifier so users know it isn't available prior to v202109-1?

Reply:
We don't typically call that out. The assumption has been what's on the website is accurate for the current release.